src/utils.ts

```diff
@@ -98,15 +98,15 @@ export const splitTokens = (tokens, core, leaf) => {
 
   const tokenList = expandTokens(tokens)
   tokenList.map((token) => {
-    const type = getTokenType(token, core, leaf)
+    const tokenType = getTokenType(token, core, leaf)
```

Comment:
I wonder if now that we have types for the tokens if this function could be confusing since it's not actually getting TypeScript types

Reply:
I agree, but I was trying to avoid making too many changes. Also, I couldn't think of a better name.